PSICompile/ILCodeGen.cs

```diff
@@ -66,8 +66,21 @@ public class ILCodeGen : Visitor {
       }
       if (w.NewLine) Out ("    call void [System.Console]System.Console::WriteLine ()");
    }
-   
-   public override void Visit (NIfStmt f) => throw new NotImplementedException ();
+
+   public override void Visit (NIfStmt f) {
+      string lab1 = NextLabel (), lab2 = NextLabel ();
+      Out ($"  {lab1}:");
+      f.Condition.Accept (this);
+      Out ($"    brfalse {lab2}");
+      f.IfPart.Accept (this);
+

```

Comment:
Unnecessary new line. It can added if you would like to separate the logic. But I think that is not the case here.